docs/reference/query-dsl/rule-query.asciidoc

```diff
@@ -78,8 +85,9 @@ GET /_search
 [[rule-query-top-level-parameters]]
 ==== Top-level parameters for `rule_query`
 
-`ruleset_id`::
-(Required, string) A unique <<query-rules-apis, query ruleset>> ID with query-based rules to match and apply as applicable.
+`ruleset_ids`::
```

Comment:
Is there something we need to document regarding the order of how the rules will be applied?

Reply:
Good catch!